### PR TITLE
beta1 blog entry: Fix CoffeeScript example

### PR DESCRIPTION
**Pull request by wmertens**
Fix the constructor call and make code more idiomatic.

### CODE REVIEW

File: docs/_posts/2015-01-27-react-v0.13.0-beta-1.md [hunk 1]
```diff
@@ -135,17 +135,19 @@ You can also use CoffeeScript classes:
...
+    super props
+    @state = count: props.initialCount
+
   tick: =>
     @setState count: @state.count + 1
+
   render: ->
-    div(onClick: @tick, 'Clicks: ', @state.count)
+    div onClick: @tick,
+      "Clicks: #{@state.count}"
```
Comment (zpao) [inline docs/_posts/2015-01-27-react-v0.13.0-beta-1.md]: This is actually different than what we have in the ES6 class above. Doing the string interpolation like this results in a single text child, where as what we do will result in 2.

The rest seems fine.

Comment (wmertens) [inline docs/_posts/2015-01-27-react-v0.13.0-beta-1.md]: @zpao I was thinking that the interpolation is closer in look/feel to the way the JSX is written and is more idiomatic. But two arguments works too of course.

Changed and squashed.

**Comment by spicyj** [comment]
@wmertens Can you sign the CLA here?

**Comment by wmertens** [comment]
Well I signed the CLA but no automated message it seems.

**Comment by spicyj** [comment]
No worries, thank you!

**Pull request merged**